run_scans.py

```diff
@@ -166,8 +176,8 @@ def launchArgs(tool, shelf, slot, link, chamber, vfatmask, scanmin, scanmax, nev
                     metavar="randoms")
   parser.add_option("--series", action="store_true", dest="series",
                     help="Run tests in series (default is false)", metavar="series")
-  parser.add_option("--stepSize", type="int", dest="stepSize", 
-                    help="Supply a step size to the latency scan from scanmin to scanmax", metavar="stepSize", default=1)
```

Comment:
why where these lines removed...?  The `stepSize` option is needed below when calling `launch(...)`.

Perhaps this was unintentional and occurred automatically during the rebase by `git`.  In any case.  It needs to be added back in.

Reply:
This option is defined in qcoptions.py and so already exists when the command "from gempython.vfatqc.qcoptions import parser" is called

Reply:
noted